releases/release-1.20/README.md

```diff
@@ -105,4 +105,4 @@ Please refer to the [release phases document](../release_phases.md).
 [1.20-blocking]: https://testgrid.k8s.io/sig-release-1.20-blocking#Summary
 
 [exception requests]: ../EXCEPTIONS.md
-[release phases document]: ../release_phases.md
\ No newline at end of file
+[release phases document]: ../release_phases.md
```

Comment:
Is this change just a newline removal?

Reply:
Hi @hasheddan 👋  I directly edited the file in Github interface, seems it has done that by itself 🤔  I will fix that 👍

Reply:
awesome! thanks @ebriand :)

Reply:
Done 👌  Thanks @hasheddan 👍